app/views/articles/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<body>
+<div class="main">
+  <div class="container">
+
+    <p>heyyyy</p>
+
+    <%= form_for(:article, url: article_new_path) do |f| %>
+
+      <% if @article.errors.any? %>
+        <div id="error_explanation">
+          <h2><%= pluralize(@article.errors.count, "error") %> prohibited this article from being saved:</h2>
+
+          <ul>
+            <% @article.errors.full_messages.each do |msg| %>
+              <li><%= msg %></li>
+            <% end %>
+          </ul>
+        </div>
+      <% end %>
+
+      <p>theeerrre</p>
```

Comment:
Hehe, I like this. Again though, did you mean to leave it in?

Reply:
While quirky and whimsical, I have elected to remove them in my most recent commit :)